### PR TITLE
Attempt to update workflow to respect the requested Zowe versions

### PR DESCRIPTION
**Pull request by awharn**
Signed-off-by: Andrew W. Harn <andrew.harn@broadcom.com>

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -25,8 +25,19 @@ jobs:
...
-      run: npm i --prod
+      id: install-node
+      run: | 
+        npm i --prod
+        npm install -g puppeteer-cli
+
+    - name: Install PIP Dependencies
+      id: install-pip
+      if: ${{ github.event.inputs.zowe-version && github.event.inputs.zowe-version != ''}}
+      run: pip3 install yq
```
Comment (t1m0thyj) [inline .github/workflows/main.yml:40]: Curious whether this step is required? `yq` should already be present in the Ubuntu 20.04 GitHub runner: https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md#tools

**Review by t1m0thyj — APPROVED** [review]
Clever approach to use `yq` 🙂

File: .github/workflows/main.yml (hunk 1 above)
Comment (awharn) [reply .github/workflows/main.yml:40]: I'll give it a try without. I didn't realize they have yq already.

Comment (t1m0thyj) [reply .github/workflows/main.yml:40]: Oops, didn't realize that is a different `yq` (v4.13.4) than the one on [PyPI](https://pypi.org/project/yq/#history) (v2.12.2 released in June).

**Pull request merged**